.docfx/Dockerfile.docfx

```
@@ -1,14 +1,14 @@
-﻿FROM nginx:1.27.3-alpine AS base
+﻿FROM --platform=$BUILDPLATFORM nginx:1.27.5-alpine AS base
 RUN rm -rf /usr/share/nginx/html/*
 
-FROM codebeltnet/docfx:2.77.0 AS build
+FROM --platform=$BUILDPLATFORM codebeltnet/docfx:2.78.3 AS build
 
 ADD [".", "docfx"]
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)